### PR TITLE
It is recommended to use emplace_back instead of push_back.

### PR DESCRIPTION
**Pull request by monster29000**
### Checklist:

- [ ] My code has been run through [clang-format](https://github.com/obsproject/obs-studio/blob/master/.clang-format).
- [ ] I have read the [**contributing** document](https://github.com/obsproject/obs-studio/blob/master/CONTRIBUTING.rst).
- [ ] My code is not on the master branch.
- [ ] The code has been tested.
- [ ] All commit messages are properly formatted and commits squashed where appropriate.
- [ ] I have included updates to all appropriate documentation.

### CODE REVIEW

**Comment by WizardCM** [comment]
Please provide documentation on your claim, fill in the PR template, follow the commit guidelines, and fix your code as it currently doesn't build on any platforms.

File: deps/json11/json11.hpp
Comment (tytan652) [inline deps/json11/json11.hpp:1]: Drop those changes, json11 is only used by code that was not migrated to nlohmann.
There is no plan to use it anymore.

File: deps/json11/json11.cpp
Comment (tytan652) [inline deps/json11/json11.cpp:1]: Drop those changes, json11 is only used by code that was not migrated to nlohmann.
There is no plan to use it anymore.

**Comment by notr1ch** [comment]
https://quuxplusone.github.io/blog/2021/03/03/push-back-emplace-back/ is required reading on this subject. Most of these changes seem unnecessary.

**Comment by Fenrirthviti** [comment]
Since this was submitted without any context, no information provided as part of the PR, and is of questionable value, I am closing this.

If the template is updated and proper justification found, we can re-open at that time.

**Pull request closed without merging**